tests/engine/src/commonTest/kotlin/GzipTest.kt

```diff
@@ -50,5 +47,12 @@ class HttpEngineTest {
     }
 
     mockServer.stop()
+    engine.dispose()
   }
-}
\ No newline at end of file
+
+  @Test
+  fun gzipTestDefault() = gzipTest(DefaultHttpEngine())
+
+  @Test
+  fun gzipTestKtor() = gzipTest(KtorHttpEngine())
```

Comment:
Any objection to have 2 test tasks that substitute the engine constructor? Like we do for integration tests and Java vs Kotlin models? I can try in a follow up PR if you're up for it

Reply:
Yes that would be nice, duplicating all test methods is annoying. Not so sure about how to do it though 😅 so you can go ahead if you have ideas 🙏